### ISSUE TITLE
2.3.0-rc1: ERROR [AbstractRotationStrategy] Cannot perform rotation at this moment.

### ISSUE THREAD

**Issue by frinnst** [Graylog2/graylog2-server#3997]
When using  "Index Size" rotation strategy (5GB in my case) my graylog-server.log fills up with these messages:
```
2017-07-14T08:58:10.087Z ERROR [AbstractRotationStrategy] Cannot perform rotation at this moment.
2017-07-14T08:58:20.100Z ERROR [AbstractRotationStrategy] Cannot perform rotation at this moment.
2017-07-14T08:58:30.092Z ERROR [AbstractRotationStrategy] Cannot perform rotation at this moment.
2017-07-14T09:10:50.080Z ERROR [AbstractRotationStrategy] Cannot perform rotation at this moment.
2017-07-14T09:11:00.188Z ERROR [AbstractRotationStrategy] Cannot perform rotation at this moment.
2017-07-14T09:11:10.087Z ERROR [AbstractRotationStrategy] Cannot perform rotation at this moment.
2017-07-14T09:11:20.100Z ERROR [AbstractRotationStrategy] Cannot perform rotation at this moment.
2017-07-14T09:11:30.085Z ERROR [AbstractRotationStrategy] Cannot perform rotation at this moment.
```
If I switch to "Index Time" for example the log becomes quiet again.

## Steps to Reproduce
1. Switch Index rotation to "Index Size"
2. tail -f /var/log/graylog-server/server.log

* Graylog Version: 2.3.0-rc1
* Elasticsearch Version: 5.5.0
* MongoDB Version: 2.6.12
* Operating System: ubuntu 16.04
* Browser version: Various

**Comment by bernd** [comment]
Thank you for the report. I am able to reproduce that and currently work on a fix.

**Comment by bernd** [comment]
@frinnst This will be fixed in the next 2.3 release candidate.
